### PR TITLE
Starting to move to oncotator MAF instead of VCF

### PR DESCRIPTION
**Pull request by LeeTL1220**
Additions to support MAF generation instead of VCF.

### CODE REVIEW

**Comment by LeeTL1220** [comment]
@davidbenjamin Have you merged your branch yet?  I don't see a PR ... apologies if I missed it.

File: scripts/mutect2_wdl/README.md [hunk 1]
```diff
@@ -64,6 +68,21 @@ TUMOR_2_BAM</TAB>TUMOR_2_BAM_INDEX</TAB>TUMOR_2_SAMPLE
 . . .
 ```
 
+- ``Mutect2_Multi.onco_ds_tar_gz`` -- (optional)  A tar.gz file of the oncotator datasources -- often quite large (>15GB).  This will be uncompressed as part of the oncotator task.  Depending on backend used, this can be specified as a path on the local filesystem of a cloud storage container (e.g. gs://...).  Typically the Oncotator default datasource can be downloaded at ``ftp://gsapubftp-anonymous@ftp.broadinstitute.org/bundle/oncotator/``.  Do not put the FTP URL into the json file.
+- ``Mutect2_Multi.onco_ds_local_db_dir`` -- (optional)  A direct path to the Oncotator datasource directory (uncompressed).  While this is the fastest approach, it cannot be used with docker unless your docker image already has the datasources in it.  For cromwell backends without docker, this can be a local filesystem path.  *This cannot be a cloud storage location*
+
+ Note:  If neither ``Mutect2_Multi.onco_ds_tar_gz`` nor ``Mutect2_Multi.onco_ds_local_db_dir`` are specified, the Oncotator task will download and uncompress for each execution.
```
Comment (SHuang-Broad) [inline scripts/mutect2_wdl/README.md]: "are specified" -> "is specified"?

File: scripts/mutect2_wdl/README.md [hunk 2]
```diff
@@ -64,6 +68,21 @@ TUMOR_2_BAM</TAB>TUMOR_2_BAM_INDEX</TAB>TUMOR_2_SAMPLE
...
+
+The following three parameters are useful for rendering TCGA MAFs using oncotator.  These parameters are ignored if ``is_run_oncotator`` is ``false``.
+- ``Mutect2_Multi.sequencing_center`` -- (optional) center reporting this variant.     Please see ``https://wiki.nci.nih.gov/display/TCGA/Mutation+Annotation+Format+%28MAF%29+Specification+-+v2.4`` for more details.   
+- ``Mutect2_Multi.sequence_source`` -- (optional)  ``WGS`` or ``WXS`` for whole genome or whole exome sequencing, respectively.  Please note that the controlled vocabulary of the TCGA MAF spec is *not* enforced.  Please see ``https://wiki.nci.nih.gov/display/TCGA/Mutation+Annotation+Format+%28MAF%29+Specification+-+v2.4`` for more details.
+- ``Mutect2_Multi.default_config_file`` -- (optional)  A configuration file that can direct oncotator to use default values for unspecified annotations in the TCGA MAF.  This help prevents having MAF files with a lot of "__UNKNOWN__" values.  An usable example is given below.  This parameter is ignored if ``is_run_oncotator`` is ``false``.  Here is an example that should work for most users:
```
Comment (SHuang-Broad) [inline scripts/mutect2_wdl/README.md]: This parameter is ignored if ``is_run_oncotator`` is ``false``.
The above sentence is duplicate, as all three are ignored as mentioned above.

File: scripts/mutect2_wdl/mutect2.wdl [hunk 3]
```diff
@@ -450,20 +460,22 @@ task oncotate_m2 {
...
-        disks: "local-disk 150 SSD"
+        memory: "3 GB"
+        bootDiskSizeGb: 12
+        disks: "local-disk 100 HDD"
         preemptible: "${preemptible_attempts}"
     }
 
     output {
-        File oncotated_m2_vcf="${entity_id}.oncotated.vcf"
+        File oncotated_m2_maf="${entity_id}.maf.annotated"
```
Comment (SHuang-Broad) [inline scripts/mutect2_wdl/mutect2.wdl]: not sure about this (not familiar with MAF format), but I would, intuitively, expect a file named "*.annotated.maf".

File: scripts/mutect2_wdl/README.md (hunk 1 above)
Comment (LeeTL1220) [inline scripts/mutect2_wdl/README.md]: Done

File: scripts/mutect2_wdl/README.md (hunk 2 above)
Comment (LeeTL1220) [inline scripts/mutect2_wdl/README.md]: Done

File: scripts/mutect2_wdl/mutect2.wdl (hunk 3 above)
Comment (LeeTL1220) [inline scripts/mutect2_wdl/mutect2.wdl]: Previous convention in cancer groups has been maf.annotated.

Comment (LeeTL1220) [inline scripts/mutect2_wdl/mutect2.wdl]: No action

**Pull request merged**